src/main/java/frc/robot/Robot.java

```diff
@@ -40,6 +41,8 @@ public void robotInit() {
     this.robotContainer = new RobotContainer();
 
     datalog.dataLogRobotContainerInit(this.robotContainer);
+
+    SmartDashboard.putData(CommandScheduler.getInstance());
```

Comment:
Is there no way to put this into dataLogRobotContainerInit? I'd like to isolate most of the logging in datalogging.java if possible, and putData calls into RobotContainer.java.

Reply:
That should be fine. I’ll try it this evening and update. I think I just followed the WIPLib example.